anime-list-master.xml

```diff
@@ -43224,7 +43224,7 @@
   <anime anidbid="15731" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>Bara Ou no Souretsu</name>
   </anime>
-  <anime anidbid="15732" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="15732" tvdbid="82788" defaulttvdbseason="0" episodeoffset="12" tmdbid="773621" imdbid="">
```

Comment:
This movie has IMDB id tt13210990, https://www.imdb.com/title/tt13210990/

Reply:
Ok I added it